plugin.yml

```diff
@@ -1,7 +1,14 @@
 name: GitRollbacks
 main: jasonwynn10\GitRollbacks\Main
-version: 0.1.0
+version: 0.2.0
 api: 3.0.0
 description: ""
 author: "jasonwynn10"
-load: STARTUP
\ No newline at end of file
+load: STARTUP
+os:
```

Comment:
Just here for consistency? Having all 3 makes this obsolete afaik

Reply:
There are more than 3 operating systems... these 3 are the ones supported.

Reply:
Hm, but afair pocketmine only checks for win, mac und linux.. Sorry if i am mistaken ^^"

Reply:
Pocketmine checks for Windows, Linux, Android, mac, iOS, and something else I can remember